plugins/jobtype/jobtypes/spark/private.properties

```diff
@@ -0,0 +1,3 @@
+jobtype.class=azkaban.jobtype.HadoopSparkJob
+
+jobtype.classpath=${hadoop.classpath}:${spark.home}/conf:${spark.home}/conf:${spark.home}/lib/spark-assembly-1.4.0.1-hadoop2.3.0.jar:${spark.home}/lib/datanucleus-api-jdo-3.2.6.jar:${spark.home}/lib/datanucleus-core-3.2.10.jar:${spark.home}/lib/datanucleus-rdbms-3.2.9.jar
```

Comment:
quick note here: there is a rogue spark-examples-1.4.0.1-hadoop2.3.0.jar that will break things if it's in the classpath, so I am unable to do ${spark.home}/lib